schema/vrs.yaml

```diff
@@ -396,27 +387,77 @@ definitions:
     required: [ "type", "gene_id" ]
 
   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-  # Range
+  # Numerics, Comparators, and Ranges
+
+  IntegerOrRange:
+    description: >-
+      A set of classes that are used to represent either single integer values
+      (such as SimpleInteger) or integer ranges (such as IntegerRange).
+    oneOf:
+      - $ref: "#/definitions/IntegerValue"
+      - $ref: "#/definitions/IntegerRange"
+    discriminator:
+      propertyName: type
 
-  Range:
+  IntegerValue:
     description: >-
-      An inclusive range of integers. A range may be unbounded by omitting either
-      min or max values. Fully unbounded ranges are not permitted (one of min or
-      max MUST be specified).
+      An abstract class for related integer-based value classes.
     type: object
+    oneOf:
+      - $ref: "#/definitions/SimpleInteger"
+      - $ref: "#/definitions/OpenInteger"
+    discriminator:
+      propertyName: type
+
+  SimpleInteger:
+    description: >-
+      A simple integer value. This is the minimal implementation of an IntegerValue
+      subclass.
     additionalProperties: false
     properties:
       type:
         type: string
-        enum: [ "Range" ]
-        default: "Range"
+        enum: [ "OpenInteger" ]
+        default: "OpenInteger"
+      value:
+        type: integer
+    required: [ "type", "value" ]
+
+  OpenInteger:
```

Comment:
Why "Open"? The enums  are >= and <=, so it's actually closed.

Reply:
Okay. This concept is directly analogous to a "half-bounded interval", but since we are using interval to mean something else in VRS, I propose we go with `HalfBoundedRange`.